pkg/reconciler/parallel/parallel.go

```diff
@@ -137,6 +144,11 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, p *v1.Parallel) pkgrecon
 		return fmt.Errorf("error removing unwanted Subscriptions: %w", err)
 	}
 
+	err := auth.UpdateStatusWithEventPolicies(featureFlags, &p.Status.AppliedEventPoliciesStatus, &p.Status, r.eventPolicyLister, v1.SchemeGroupVersion.WithKind("Parallel"), p.ObjectMeta)
```

Comment:
```suggestion
	err := auth.UpdateStatusWithEventPolicies(featureFlags, &p.Status.AppliedEventPoliciesStatus, &p.Status, r.eventPolicyLister, v1.Kind("Parallel"), p.ObjectMeta)
```

Reply:
```
cannot use v1.Kind("Parallel") (value of type "k8s.io/apimachinery/pkg/runtime/schema".GroupKind) as "k8s.io/apimachinery/pkg/runtime/schema".GroupVersionKind value in argument to auth.UpdateStatusWithEventPolicies
```
vi.Kind returns `schema.GroupKind` but the arg type is `schema.GroupVersionKind`

Reply:
ah my bad, sorry for the confusion!